components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/AsyncApiParser.java

```diff
@@ -45,6 +49,176 @@
 
 public class AsyncApiParser extends APIDefinition {
 
+    String metaSchema = "{\n" +
```

Comment:
can we put this to a file in the resource file to jar without hardcoding inside code?